action.yml

```diff
@@ -11,18 +11,19 @@ inputs:
   secrets:
     description: "One or more secret Ids to retrieve and the corresponding GitHub environment variable name to set"
     required: true
+  cloud_region:
+    description: "(Optional) The Bitwarden server region to use if cloud-hosted service is used. Either 'us' or 'eu'"
+    required: false
+    default: "us"
   base_url:
     description: "(Optional) For self-hosted bitwarden instances provide your https://your.domain.com"
     required: false
-    default: ""
```

Comment:
the GH actions dev documentation says that there should always be a default when the `required` is `false`, even if it is just empty string.
I suggest we do the same for `identity_url` and `api_url`